app/javascript/util/log_event.js

```diff
@@ -14,33 +16,43 @@ export const logEvent = (eventName: string, payload: any) => {
 
   if (window.TRACK_USER_ACTIONS) window.mixpanel.track(eventName, opts);
 
-  if (window.ga) logToGa(eventName, payload);
+  if (window.ga) logToGa(eventName, ...payload);
 };
 
-const getEventData = (eventName: string, data: any) => {
+const getEventData = (eventName: string, ...data: any) => {
   switch (eventName) {
```

Comment:
Can you talk a bit about why you've turned `data` into a rest parameter (`...data`)?

Reply:
@osahyoun some events (`ee.emit(...)`) have more than one parameter, and we were losing that extra data previously as we were only ever using the first parameter.

Reply:
Thanks